cibyl/utils/dicts.py

```diff
@@ -47,7 +44,7 @@ def subset(dictionary, keys):
     return result
 
 
-def nsubset(dictionary, keys):
+def nsubset(dictionary, keys: list) -> dict:
```

Comment:
This function is still missing the type hint for dictionary, and has it annotated in the docstring

Reply:
Sure, I agree that there is no point in specifying further than dict, but in this function it's still in the docstring and not yet in the function signature for `dictionary` argument. It's nitpicking, but just for consistency